webapp/app.py

```diff
@@ -39,9 +39,10 @@
 
 
 def get_jenkins_job(action):
+    # To be changed after QA
     return {
-        "opened": "webteam/job/start-demo",
-        "synchronize": "webteam/job/start-demo",
+        "opened": "webteam/job/start-demo-test",
+        "synchronize": "webteam/job/start-demo-test",
```

Comment:
Don't forget to change the Jenkins job name back.

Reply:
Done! Thanks a bunch